official/docs/node/addresses/create_addresss.js

```diff
@@ -0,0 +1,15 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+const address = new api.Address({
+    street1: '417 MONTGOMERY ST',
+    street2: 'FLOOR 5',
+    city: 'SAN FRANCISCO',
+    state: 'CA',
+    zip: '94104',
+    country: 'US',
+    company: 'EasyPost',
+    phone: '415-123-4567',
+  });
+  
+  console.log(address);
```

Comment:
```suggestion
console.log(address);
```
extra spaces, please check this on other files as well